CONTRIBUTING.md

```diff
@@ -70,9 +70,19 @@ following rules before you submit a pull request:
    [Utilities for Developers](http://scikit-learn.org/dev/developers/utilities.html#developers-utils)
    page.
 
--  If your pull request addresses an issue, please use the pull request title
-   to describe the issue and mention the issue number in the pull request description. This will make sure a link back to the original issue is
-   created.
+-  Give your pull request a helpful title that summarises what your
+   contribution does. In some cases "Fix <ISSUE TITLE>" is enough.
+   "Fix #<ISSUE NUMBER>" is not enough.
+
+-  Often pull requests resolve one or more other issues (or pull requests).
+   If merging your pull request means that some other issues/PRs should
+   be closed, you should `use keywords to create link to them
+   <https://github.com/blog/1506-closing-issues-via-pull-requests/>`_
+   (e.g., ``Fixes #1234``, ``Closes #1234``; multiple issues/PRs are
```

Comment:
When I said "use only Fix or only Close" I meant everywhere. Can you fix it here too and in all the other places this happens. Just use "Fix" or just use "Close" but not both.